interop-java/jvm/src/test/scala/scalaz/zio/interop/javaconcurrent.scala

```diff
@@ -25,27 +25,29 @@ import scala.concurrent.ExecutionException
 
 object javaconcurrent {
 
-  implicit class IOObjJavaconcurrentOps(private val taskObj: Task.type) extends AnyVal {
+  implicit class IOObjJavaconcurrentOps(private val ioObj: IO.type) extends AnyVal {
```

Comment:
How about we move these changes to a separate pull request?

Reply:
Either way - let's keep this on the `Task` object, as all Java futures may fail with a throwable.

Reply:
Again :(